examples/inverse/covariance_whitening_dspm.py

```diff
@@ -1,188 +0,0 @@
-"""
```

Comment:
@agramfort I think removing this example is the only thing that needs to be checked/confirmed before I visually confirm the others and merge

Basically I think this is mostly redundant now with the other existing tutorial/examples. Plus we can refer people to the paper for more info. WDYT? (To me it doesn't seem worth ~2 min of CI time for this example, but I don't see a clean way to make it faster.)